circle.yml

```diff
@@ -1,11 +1,15 @@
-machine:
-    services:
-        - docker
-
-dependencies:
-    override:
-        - echo "Ignore CircleCI detected dependencies"
-
-test:
-    override:
-        - docker build -t steemit/steemit.com .
+version: 2
+jobs:
+    build:
+        docker:
+            - image: circleci/openjdk:jdk-node-browsers
+        steps:
+            - checkout
+            - setup_remote_docker:
+                docker_layer_caching: true
+            - run: docker build -t steemit/condenser:$CIRCLE_BRANCH .
+            - run: docker build -t steemit/condenser-blackboxtest:$CIRCLE_BRANCH blackboxtest/
+            - run: |
+                docker run -ditp 8080:8080 --name condenser steemit/condenser:$CIRCLE_BRANCH
+                sleep 10 # let condenser start
```

Comment:
Did you try the circleci 2.0 workflows with dependencies? Seems like we should be able to run the builds in parallel using that

https://circleci.com/docs/2.0/workflows/

Reply:
It would also allow us to specify in GitHub that only the main docker build step is a requirement for merging so that a blackboxtest failure won't block

Reply:
sounds cool, will try it out!

Reply:
so i set up a basic workflow; the fancy thing to do would be to refactor our existing Dockerfile into steps for `yarn install`, `unit & integration tests`, and `prod build` -- the latter two can run in parallel, and then blackboxtest would be dependent on prod build.